pkg/cmd/tknpac/create/repository.go

```diff
@@ -76,15 +76,18 @@ func repositoryCommand(run *params.Run, ioStreams *cli.IOStreams) *cobra.Command
 			}
 
 			var providerName string
-			_, installationNS, err := bootstrap.DetectPacInstallation(ctx, createOpts.pacNamespace, run)
+			installed, installationNS, err := bootstrap.DetectPacInstallation(ctx, createOpts.pacNamespace, run)
+			if !installed {
+				return fmt.Errorf("pipelines-as-code is not installed in the cluster")
+			}
 			if err != nil {
-				return err
+				// if we don't have access to the pipelines as code info configmap (ie acl, don't let you)
+				// we won't try to update it
```

Comment:
How about adding some info log here  :thinking: 
Because if we just generate template and return how end use will get to know what happened